libraries/DataFlash/DataFlash_Block.h

```diff
@@ -16,7 +16,6 @@ class DataFlash_Block : public DataFlash_Backend
         DataFlash_Backend(front, writer) { }
 
     // initialisation
```

Comment:
Thought about it at the time.  Determined "CardInserted()" was near enough to initialisation for me :-)